.github/workflows/ci.yml

```diff
@@ -178,8 +177,8 @@ jobs:
           - { compiler: GNU10,  CC: gcc-10,   CXX: g++-10,     packages: gcc-10 g++-10 }
           - { compiler: GNU11,  CC: gcc-11,   CXX: g++-11,     packages: gcc-11 g++-11 }
           #- { compiler: LLVM9,  CC: clang-9,  CXX: clang++-9,  packages: clang-9 libomp-9-dev libclang-common-9-dev llvm-9-dev clang++-9 libc++-9-dev lld-9}
-          - { compiler: LLVM10, CC: clang-10, CXX: clang++-10, packages: clang-10 libomp-10-dev libclang-common-10-dev llvm-10-dev clang++-10 libc++-10-dev lld-10}
-          #- { compiler: LLVM11, CC: clang-11, CXX: clang++-11, packages: clang-11 libomp-11-dev libclang-common-11-dev llvm-11-dev clang++-11 libc++-11-dev lld-11}
+          #- { compiler: LLVM10, CC: clang-10, CXX: clang++-10, packages: clang-10 libomp-10-dev libclang-common-10-dev llvm-10-dev clang++-10 libc++-10-dev lld-10}
```

Comment:
Could even bump to or also include LLVM12 as it is also available?

Reply:
If it becomes avail, we'll bump to LLVM12, however when doing this change the LLVM available consistently was 11.

Reply:
https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md lists 12.0 as available

Reply:
Hmmmm... I'll check later after work...

Reply:
However, 11 seems to be the "default": https://github.com/actions/virtual-environments/blob/126faeedce84fedfeef6e525691aec3b72427a78/images/linux/toolsets/toolset-2004.json#L247

Reply:
wouldya know... LLVM12 it is!